### DISCUSSION TITLE
Websites not loading properly since a few days

### DISCUSSION THREAD

**Discussion by esansonetti311** [Lennolium/PrivacyShield#22] [Questions and Answers] [answered]
Hello

Since a few days (about a week or so) Websites are not loading properly anymore if I activate this filter. The Website just shows blanc with text with almost no pictures. 

Is there a fix for that? 

Here is an example: https://www.admin.ch/gov/de/start.html

<img width="805" alt="Issue" src="https://github.com/Lennolium/PrivacyShield/assets/20194192/b9ad6b2c-7ea0-4421-bca6-034acfa192a7">

**Comment by Lennolium** [comment]
Hello @esansonetti311!

First thank you for using PrivacyShield, I appreciate your feedback.

Regarding your issue, I will investigate the source of the false website rendering. The example website you gave seems to load its resources/stylesheets maybe externally and these domains are getting blocked. Could you tell me which version (bronze, silver, ...) of PrivacyShield you are using so I can fix it in the next release.

**Quick Fix:** You could change to a 'lower' version (if you are now using platinum, try gold, check if website is working, if not try silver, and so on ...).

**Comment by esansonetti311** [comment]
Hi!

Thank you for your fast reply. I am using Platinum, but now I downgraded to Gold as I thought maybe Platinum was too aggressive (though I hadn’t any problems before). 
Another site where it doesn’t work are computerbase.de or faz.net, just to name you a few.

How can I downgrade to an older version? So I can try if the problem persists.

Cheers

**Comment by Lennolium** [comment]
Hey,

With downgrading I exactly meant, that you go from platinum to gold, and maybe to silver.
If you really want to try the earlier release version of gold (v1.1.1) use the following file/link: 
[PrivacyShield-gold (v1.1.1)](https://raw.githubusercontent.com/Lennolium/PrivacyShield/c34d4a83287237aae0a31115b73381b7b77e5d9b/Filters/PrivacyShield-gold.txt)

Thanks for listing these domains. If you have any other non-working websites, feel free to post them. I will whitelist them.
Future plans include a website where users can input domains and they get automatically whitelisted from the lists.

Have a good day :)

**Comment by esansonetti311** [comment]
Hey

Thank you! Unfortunately, it happens to almost every second website =/

In fact, with your v1.1.1 Gold script the websites work as intended! Thank you for that!

Could you be so kind and maybe provide the Platinum one?

Cheers

**Comment by Lennolium** [comment, marked as answer]
Alright thanks for letting me know, I will take a look 👍🏻 

Here are the older versions:

> [bronze](https://raw.githubusercontent.com/Lennolium/PrivacyShield/c34d4a83287237aae0a31115b73381b7b77e5d9b/Filters/PrivacyShield-bronze.txt)
> [silver](https://raw.githubusercontent.com/Lennolium/PrivacyShield/c34d4a83287237aae0a31115b73381b7b77e5d9b/Filters/PrivacyShield-silver.txt)
> [gold](https://raw.githubusercontent.com/Lennolium/PrivacyShield/c34d4a83287237aae0a31115b73381b7b77e5d9b/Filters/PrivacyShield-gold.txt)
> [platinum](https://raw.githubusercontent.com/Lennolium/PrivacyShield/1aabf2f9b81de0fdb5696006231fd7160794f07b/Filters/PrivacyShield-platinum.txt)
> [diamond](https://raw.githubusercontent.com/Lennolium/PrivacyShield/1aabf2f9b81de0fdb5696006231fd7160794f07b/Filters/PrivacyShield-diamond.txt)

Hope this helps,
Your Lennart

**Comment by esansonetti311** [comment]
Thanks!

Cheers
